lld/test/ELF/riscv-tlsdesc.s

```diff
@@ -29,6 +29,12 @@
 # RUN: ld.lld -e 0 -z now a.32.o c.32.so -o a.32.ie
 # RUN: llvm-objdump --no-show-raw-insn -M no-aliases -h -d a.32.ie | FileCheck %s --check-prefix=IE32
 
+# RUN: llvm-mc -triple=riscv64 --position-independent -filetype=obj d.s -o d.64.o
```

Comment:
Delete --position-independent, which is only relevant for mips.

Reply:
Thanks for pointing that out.